packages/form-types-bundle/composer.json

```diff
@@ -2,12 +2,7 @@
     "name": "shopsys/form-types-bundle",
     "type": "library",
     "description": "Shopsys Framework custom Symfony form types",
-    "keywords": [
-        "form",
-        "form type",
-        "Shopsys Framework",
-        "SSFW"
-    ],
+    "keywords": ["form", "form type", "Shopsys Framework", "SSFW"],
```

Comment:
https://github.com/shopsys/shopsys/commit/9c9c938a78c9c989f31884ec35a60b52e715d93c#diff-6963f8b39cb568c5818c8821fdbff980

Reply:
yes, this is weird, it was formatted automatically this way? I changed it in the linked commit to avoid such changes :disappointed:

Reply:
This was formatted (and committed) automatically in one of the workers.

Reply:
seems like the behavior of the `Symplify\MonorepoBuilder\FileSystem\JsonFileManager` has changed then...